lambda/conftest.py

```diff
@@ -1,28 +1,29 @@
 import json
+from typing import Dict, List, Union
```

Comment:
I tend to allow tests to be untyped, what do you think?

Reply:
I think we should type everything but I'm happy for that to be best effort.